core/src/test/java/org/everit/json/schema/loader/JsonValueTest.java

```diff
@@ -1,27 +1,5 @@
 package org.everit.json.schema.loader;
 
-import static java.util.Arrays.asList;
```

Comment:
Is this import reordering necessary? If there is no significant change then can you please revert it?

Reply:
I have to admit that I just let the IDE autofixing the unused imports of 
import java.util.ArrayList;
import java.util.Arrays;

I can revert the changes and just remove the unused imports